src/frontend/shortlist/src/components/layout/signup_login/Signup.vue

```diff
@@ -79,8 +79,20 @@ export default {
         <input type="password" v-model="passwordVerify_signup" />
       </label>
       <button type="submit" class="btn btn-outline-dark">Sign up</button>
+      <p class="instructions" id="small">or</p>
+      <button
+        id="signupWithGoogle"
+        @click.prevent="loginWithSSO"
+        class="btn btn-outline-dark"
+      >
+        Sign Up with Google
+      </button>
```

Comment:
Are these changes you merged from the demo of last week?

Reply:
No. I found out someone left logInWithSSO but took out the signUpWithSSO, so I added it back, as I commented there: if there is no signup with SSO, how can there be login with SSO?